resources/translations/en.edn

```diff
@@ -331,6 +331,7 @@
          :events "Events"
          :failed "Failed"
          :has-accepted-licenses "Terms of use accepted."
+         :invalid-attachment "Attachment not acceptable"
```

Comment:
Would be nice to list here the allowed types. 

Also isn't it more like "Attachment type not acceptable"? Also there is now attachment being acceptable and allowed file extensions. How about unifying and using "Acceptable file types" and "Attachment type not acceptable".